src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -5254,7 +5254,7 @@ nuget update -Self</value>
     <value>A list of package sources to use as fallbacks for this command.</value>
   </data>
   <data name="CommandMSBuildVersion" xml:space="preserve">
-    <value>Specifies the version of MSBuild to be used with this command. Supported values are 4, 12, 14. By default the MSBuild in your path is picked, otherwise it defaults to the highest installed version of MSBuild.</value>
+    <value>Specifies the version of MSBuild to be used with this command. Supported values are 4, 12, 14, 15.1, 15.3, 15.4, 15.5, 15.6, 15.7, 15.8, 15.9. By default the MSBuild in your path is picked, otherwise it defaults to the highest installed version of MSBuild.</value>
```

Comment:
As this will ship in 5.0 (by which 16.0 will be release), can you please add that as well. 

We need a better way of communicating this (an auto-gened list would work), but versions getting skipped such as 15.2 is a problem.

It's fine for now.

Reply:
16 added.